oci/caps/utils.go

```diff
@@ -16,18 +16,6 @@ func init() {
 	if last == capability.Cap(63) {
 		last = capability.CAP_BLOCK_SUSPEND
 	}
-	if last > capability.CAP_AUDIT_READ {
-		// Prevents docker from setting CAP_PERFMON, CAP_BPF, and CAP_CHECKPOINT_RESTORE
-		// capabilities on privileged (or CAP_ALL) containers on Kernel 5.8 and up.
-		// While these kernels support these capabilities, the current release of
-		// runc ships with an older version of /gocapability/capability, and does
-		// not know about them, causing an error to be produced.
-		//
-		// FIXME remove once https://github.com/opencontainers/runc/commit/6dfbe9b80707b1ca188255e8def15263348e0f9a
-		//       is included in a runc release and once we stop supporting containerd 1.3.x
-		//       (which ships with runc v1.0.0-rc92)
-		last = capability.CAP_AUDIT_READ
-	}
 	for _, cap := range capability.List() {
```

Comment:
I think we should wait for https://github.com/containerd/containerd/pull/5017 and use the function from containerd. Otherwise Docker21-in-Docker20 won't work.

Reply:
Actually was thinking of creating a PR to move the go capabilities package (with git history) to github.com/opencontainers/runtime-spec), so that both containerd, moby, and runc (possibly cri-o) can use of the same package WDYT?

Reply:
Do you mean moving https://github.com/syndtr/gocapability to the runtime-spec repo? I think that should be discussed with the author of the package. Anyway, I'm not sure the runtime-spec repo can contain non-spec codes.

Reply:
>  think that should be discussed with the author of the package

For sure.

> Anyway, I'm not sure the runtime-spec repo can contain non-spec codes.

Ah, yes, I thought there were some utility functions in the repo, but looks like there aren't.

The constants themselves could perhaps make sense, similar to `SCMP_ARCH` arches are defined there. idk. Basically, I think it'd be good if there was a central place for code like this (could also still be in the   https://github.com/syndtr/gocapability repository)

Reply:
FYI: kernel.org hosts Go code for caps: https://pkg.go.dev/kernel.org/pub/linux/libs/security/libcap/cap#Value

Reply:
Discussing in the maintainers meeting; let's split this one into two separate PRs, and start with just updating runc